### DISCUSSION TITLE
bug

### DISCUSSION THREAD

**Discussion by GuqingX** [GeForceLegend/Minecraft-3D-Default#17] [General]
大佬考虑支持IRIS吗 
 另 1.4.2版本  1.18.1 烤猪排还有白色边框线条  无optifine 无光影

**Comment by GeForceLegend** [comment]
> 大佬考虑支持IRIS吗 另 1.4.2版本 1.18.1 烤猪排还有白色边框线条 无optifine 无光影

一切原版扁平物品模型出问题直接删除assets/minecraft/models/item/generated.json
作为一个资源包，使用的是Optifine的资源包接口，而IRIS只是一个单纯的着色器加载器，不存在需要支持的东西，倒是IRIS的依赖Sodium因为爆改原版光照会让一些模型的效果炸掉
